tests/test_utils/perf/dags/elastic_dag.py

```diff
@@ -175,13 +175,13 @@ class DagShape(Enum):
                     f"DAGS_COUNT={dag_no}_of_{DAG_COUNT}",
                     f"TASKS_COUNT=${TASKS_COUNT}",
                     f"START_DATE=${START_DATE_ENV}",
-                    f"SCHEDULE_INTERVAL=${SCHEDULE_INTERVAL_ENV}",
+                    f"schedule=${SCHEDULE_INTERVAL_ENV}",
```

Comment:
Not quite sure what this one is for, it’s not actually DAG arguments.

Reply:
i guess it's to generate a dag_id. i'll make it uppercase.